apps/react-workshop/src/ComboBox.stories.tsx

```diff
@@ -519,16 +522,28 @@ export const MultipleSelect = () => {
     'AX',
   ]);
 
+  const [clearFilterOnOptionToggle, setClearFilterOnOptionToggle] =
+    React.useState(true);
+
   return (
-    <ComboBox
-      options={options}
-      inputProps={{ placeholder: 'Select a country' }}
-      multiple
-      value={selectedOptions}
-      onChange={(selected, event) => {
-        console.log(event.value + ' ' + event.type);
-        setSelectedOptions(selected);
-      }}
-    />
+    <Flex flexDirection='column' alignItems='stretch'>
+      <Checkbox
+        checked={clearFilterOnOptionToggle}
+        onChange={(e) => setClearFilterOnOptionToggle(e.target.checked)}
+        label='clearFilterOnOptionToggle'
+      />
+      <Divider />
```

Comment:
There's some subjectiveness here, but I think this checkbox and divider should be:
1. defined using a decorator
2. be below the combobox and hidden from the visual test

It would be nice to keep the test image unchanged.